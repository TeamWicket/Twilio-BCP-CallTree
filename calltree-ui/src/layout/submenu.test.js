import SubMenu from './submenu';
import React from 'react';
import { shallow } from 'enzyme';

describe("SubMenu", () => {
    it("should match snapshot of SubMenu", () => {
        expect(shallow(<SubMenu />)).toMatchSnapshot();
    })
})
