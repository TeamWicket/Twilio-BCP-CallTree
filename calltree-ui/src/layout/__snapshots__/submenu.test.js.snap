// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubMenu should match snapshot of SubMenu 1`] = `ShallowWrapper {}`;
